pkg/publish/default.go

```diff
@@ -120,18 +124,73 @@ func pushResult(tag name.Tag, br build.Result, opt []remote.Option) error {
 		return err
 	}
 
+	// writePeripherals implements walk.Fn
+	writePeripherals := func(_ context.Context, se oci.SignedEntity) error {
```

Comment:
Can we use the context passed in here when `remote.Write`ing at least? We might also want to pipe the ctx through to SBOM generation so that interrupting `ko` cancels the `go version -m` call.

Reply:
It looks like the context we would thread through is already part of the `opts` being passed in.

Reply:
LMK if you want me to do it again for good measure!